source/fundamentals/crud.txt

```diff
@@ -1,13 +1,26 @@
-
-====
-CRUD
-====
+===============
+CRUD Operations
+===============
 
 .. default-domain:: mongodb
 
 .. toctree::
-   :titlesonly:
-   :maxdepth: 1
+   :caption: CRUD Operations
+
+   /fundamentals/crud/read-operations
+   /fundamentals/crud/write-operations
+
+CRUD (Create, Read, Update, Delete) operations allow you to work with
+the data stored in MongoDB.
+
+The CRUD operation documentation is categorized in two sections:
```

Comment:
I think the preposition for categorized should be into if it's acting on the subject (CRUD operation documentation). If the subject was "two sections", you could "categorize [each] in two sections", but I don't think that is the correct description.

```suggestion
The CRUD operation documentation is categorized into two sections:
```